app/src/main/kotlin/com/mapbox/vision/teaser/OnBackPressedListener.kt

```diff
@@ -0,0 +1,6 @@
+package com.mapbox.vision.teaser
+
+interface OnBackPressedListener {
+
+    fun onBackPressed(): Boolean
+}
```

Comment:
nit - new line in the end

Reply:
ktlint catches it

Reply:
Is ktlint applied to Teaser? I remember I've tried it once and it had a lot of errors

Reply:
fixed

Reply:
@yuryybk have just noted, yep it's [set up](https://github.com/mapbox/mapbox-vision-android-teaser/blob/ybk_common_teaser_app/app/build.gradle#L7) but no CI checker(even no CI set up :trollface:)